src/responder/kcm/kcmsrv_ops.h

```diff
@@ -29,6 +29,22 @@
 #include "util/sss_iobuf.h"
 #include "responder/kcm/kcmsrv_pvt.h"
 
+/* The initial packet size, which can later be grown up to KCM_PACKET_MAX_SIZE.
+ * The kernel will allocate whole pages even if less is requested, so it
+ * doesn't help to allocate less than one page.
+ * We should be using `sysconf(_SC_PAGE_SIZE)` to determine the page size, but
+ * that would imply calling the function each time. So we manually force the
+ * value.
+ */
+#define KCM_PACKET_INITIAL_SIZE 4096
+
+/* The maximum length of a request or reply as defined by the RPC
+ * protocol. This is the same constant size as MIT KRB5 uses
+ * This limit comes from:
+ * https://github.com/krb5/krb5/blob/master/src/lib/krb5/ccache/cc_kcm.c#L53
```

Comment:
Line number changed, btw.
Better to use perm link like https://github.com/krb5/krb5/blob/c20251dafd6120fa08c76b19315cb9deb1a1b24e/src/lib/krb5/ccache/cc_kcm.c#L54

Reply:
Right.